packages/jest-environment-node/src/index.ts

```diff
@@ -24,7 +21,7 @@ type Timer = {
 class NodeEnvironment implements JestEnvironment {
   context: Context | null;
   fakeTimers: LegacyFakeTimers<Timer> | null;
```

Comment:
could rename this to `legacyFakeTimers`, not sure it's worth the churn for custom envs, tho?